tests/test_trello_bookmarks.py

```diff
@@ -165,11 +165,11 @@ def test_run(self):
             self.assertEqual(record_count, first_record_count_by_stream[stream])
             # self.assertEqual(record_count, 1)
 
-        for stream in self.expected_incremental_sync_streams():
-            record_count = second_record_count_by_stream.get(stream, 0)
-            # We aren't generating data between the two syncs, and the
-            # bookmark should be a little behind 'now', so the second sync
-            # should return no data
-            self.assertEqual(record_count, 0)
+        # for stream in self.expected_incremental_sync_streams():
+        #     record_count = second_record_count_by_stream.get(stream, 0)
+        #     # We aren't generating data between the two syncs, and the
+        #     # bookmark should be a little behind 'now', so the second sync
+        #     # should return no data
+        #     self.assertEqual(record_count, 0)
```

Comment:
Why is this commented out?

Reply:
this originally tested that for actions on the second sync 0 records will come through. Now that we have a lookback window, records will come through on the second sync. This will be fixed when we add more extensive testing for the lookback window.